CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Thanks for helping make the Twitch CLI better! 
```

Comment:
Shouldn't it be `go-blueprint` instead on `Twitch CLI`?

Reply:
@SputNikPlop Yeah this should say go-blueprint